ui/app/styles/helper-classes/spacing.scss

```
@@ -147,6 +143,10 @@
   margin-bottom: $spacing-xxl;
 }
 
+.has-top-margin-xxs {
+  margin: $spacing-xxs 0;
```

Oh, this is actually misnamed (or doing the wrong thing, take your pick). It's adding spacing to both the top and bottom as is
```suggestion
  margin-top: $spacing-xxs;
```

---

Oh thank you - I quickly glanced at this and moved thinking I was cleaning things up 😓  I didn't even notice that `0`

I'll put back and adjust the class to make it clearer!